flask_wiki/static/js/wiki.js

```
@@ -93,6 +71,17 @@ $(document).ready(function () {
   });
 });
 
+function copyToClipboard(text) {
+  navigator.clipboard.writeText(text).then(
+    () => {
+      $('.toast').toast('show');
+    },
+    () => {
+      console.log('Problem when copying the link to the clipboard.');
```

I'm not a fan of putting a console here.
A silent error?